dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/DefaultEnrollmentService.java

```diff
@@ -167,7 +167,14 @@ public Page<Enrollment> getEnrollments(
   public RelationshipItem getEnrollmentInRelationshipItem(@Nonnull UID uid) {
     Enrollment enrollment;
     try {
-      enrollment = getEnrollment(uid);
+      enrollment =
+          getEnrollment(
+              uid,
+              EnrollmentParams.TRUE
```

Comment:
Can this conflict with a `fields` param the users can send when for example requesting `/tracker/trackedEntities`? In my mind I would expect it to be something like `enrollmentParams.withIncludeRelationships(false)` passing in the params 🤔 not 100% sure though. With this it looks to me as if they would always get the fully expanded enrollment but might actually want something more minimal.

Reply:
That is not an issue for the user because the field filtering will happen in the controller with the right fields.
It is an optimization that we should add (I thought about the case but forgot to put it in the description).
I will add a TODO for [`DHIS2-18883`](https://dhis2.atlassian.net/browse/DHIS2-18883)